pkg/dhclient/dhcp4.go

```diff
@@ -107,7 +123,9 @@ func (p *Packet4) Lease() *net.IPNet {
 }
 
 var (
-	ErrNoBootFile       = errors.New("no boot file name present in DHCP message")
+	//ErrNoBootFile represents that no pxe boot file was found
```

Comment:
nit: space it and end with punctuation

```go
var (
  // ErrNoBootFile represents that no PXE boot file was found.
  ErrNoBootFile =

  // ErrNoServerHostName ...
  ErrNoServerHostName = ...
```

(did gofmt not do the spacing on its own?)

Reply:
Full disclosure, I use `vim-go` which does my formatting (including gofmt), it did not add spaces.  I will do so now.